app/shared/footer/FooterContent.spec.js

```diff
@@ -97,29 +97,29 @@ describe('shared/footer/FooterContent', () => {
   describe('When Tampere customization is used', () => {
     let content;
 
-    before(() => {
+    beforeAll(() => {
       simple.mock(customizationUtils, 'getCurrentCustomization').returnWith('TAMPERE');
       content = getWrapper();
     });
 
-    after(() => {
+    afterAll(() => {
       simple.restore();
     });
 
     it('contains feedback link', () => {
       const feedbackLink = content.find(FeedbackLink);
-      expect(feedbackLink.length).to.equal(1);
+      expect(feedbackLink.length).toBe(1);
     });
 
-    it('renders texts for Tampere', () => {
-      const texts = content.find('p').text();
-      expect(texts).to.contain('Footer.tampereText');
-    });
+    // it('renders texts for Tampere', () => {
+    //   const texts = content.find('p').text();
+    //   expect(texts).to.contain('Footer.tampereText');
+    // });
 
```

Comment:
This could be removed instead of commenting out since the related link does not exist.